amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/HierarchicalShardSyncerTest.java

```diff
@@ -474,13 +476,19 @@ public void testCheckAndCreateLeasesForShardsWithEmptyShardList() throws Excepti
     }
 
     @Test
-    public void testCheckAndCreateLeasesForNewShardsAtTrimHorizon() throws Exception {
-        testCheckAndCreateLeaseForShardsIfMissing(constructShardListForGraphA(), INITIAL_POSITION_TRIM_HORIZON);
+    public void testCheckAndCreateLeasesForNewShardsAtTrimHorizonWithPartialLeaseTable() throws Exception {
```

Comment:
This is a case of empty lease table. Can we add partial lease table test cases?

Reply:
Good catch, will add partials to these